api/api.go

```diff
@@ -120,12 +125,36 @@ func (h *httpHandler) initRouter() {
 	}
 }
 
-func makeHandler(ctrl libnetwork.NetworkController, fct processor) http.HandlerFunc {
+const (
+	// only one Accept value for now
+	acceptRegex  = `^application/vnd\.libnetwork\.v\d\.\d\+json$`
```

Comment:
As of now, I am enforcing only one Accept-Header or no Accept-Header (which will fall back to the current version and default data type which is json)
But I can make the change to support the more common "application/json", instead of no values, it looks actually more coherent. Thanks.

Reason why I wanted to support no Accept-Header value was more for convenience  for testing, but it is not really important, browser all have rest client plugins.

Consider these API are for our dnet cli client (standalone or plugged into docker cli). It's a very selective use case and controlled environment, and client need to know which content-types are available when talking to a server anyway. Therefore I do not see an issue here in selectively supporting our custom type.